crates/router/src/core/payment_methods/cards.rs

```diff
@@ -2101,10 +2101,19 @@ pub async fn list_payment_methods(
             .to_not_found_response(errors::ApiErrorResponse::PaymentNotFound)?;
     }
 
+    // Check for `use_billing_as_payment_method_billing` config under business_profile
+    // If this is disabled, then the billing details in required fields will be empty and have to be collected by the customer
+    let billing_address_for_calculating_required_fields = business_profile
+        .as_ref()
+        .and_then(|business_profile| business_profile.use_billing_as_payment_method_billing)
+        .unwrap_or(true)
```

Comment:
```suggestion
        .unwrap_or(SOME_DEFAULT_CONST)
```

Can we make this default value a const and use it wherever unwrap_or() is called on `business_profile.use_billing_as_payment_method_billing`?